### ISSUE TITLE
CMS options

### ISSUE THREAD

**Issue by lmkirvan** [kirvanai/Gainesville-Quakers#3]
We should talk through the CMS options. 

The list here:   https://jamstack.org/headless-cms/

is pretty comprehensive (as far as I can tell) 

This one:
 https://jamstack.org/headless-cms/decap-cms/

seems like it might be the best fit based on quickly scanning the docs.  One of us should take a slightly deeper dive and see if it makes sense. It does seem designed to work with static sites and netlify, which is good.

**Comment by kirvanai** [comment]
Alright I'll do a deep dive into that in the next day or two, because it's something I'm going to need to know regardless. From what I understand, it seems like hosting the site and using a CMS is what will allow the last couple bits of the site to work? As far as setting up the contact form and newsletter goes.

**Comment by lmkirvan** [comment]
yeah that's it really. Finishing moving the basic content over is boring but pretty easy. we can do the migration of old content after we get something working and stable, there's no rush on the migration part though.
